packages/react-accordion/src/components/Accordion/useAccordion.ts

```diff
@@ -32,13 +40,17 @@ export const useAccordion = (
   });
 
   return {
-    ...rest,
-    ref,
     multiple,
     collapsible,
     navigable,
     openItems,
     requestToggle,
+    root: resolveShorthand(rest, {
+      required: true,
+      defaultProps: {
+        ref,
+      },
+    }),
```

Comment:
In RFC we agreed to not use `resolveShorthand()` for this, can you please clarify why it is used there?

Reply:
I guess for the amount of modifications I got carried away. And since there was no reason for `getNativeElementProps` in this case I just opted for `resolveShorthand` in case.

Reply:
Definitely not necessary